docs/products/postgresql/howto/postgresql-debezium-cdc.rst

```diff
@@ -0,0 +1,192 @@
+Handling PostgreSQL® node replacements when using Debezium for Change Data Capture
```

Comment:
Please check the warning, we should have either titles in sentence case or, if there's an exception, we need to add the exception (like Debezium or Change Data Capture) to `.github/vale/styles/Aiven/capitalization_headings.yml`

Reply:
also the sentence should be in present case, i would suggest

```
Handle PostgreSQL® node replacements when using Debezium for Change Data Capture
```